conference.js

```diff
@@ -845,7 +845,7 @@ export default {
             // This will only modify base/media.audio.muted which is then synced
             // up with the track at the end of local tracks initialization.
             muteLocalAudio(mute);
-            this.setAudioMuteStatus(mute);
+            this.updateAudioIconEnabled();
```

Comment:
Do we even need to call it here? What is the purpose of checking here if the audio icon needs to be enabled or disabled.